tests/testerina-integration-test/src/test/resources/project-based-tests/module-execution-flow-tests/modules/moduleA/main.bal

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
```

Comment:
Since this will pull the module from the central, shall we revert ece77dc0a53e6c2ceee8c808978dd71ec8761e91 or try to fix this correctly?

Reply:
Restore the earlier changes. We can address the `ballerina/io` usages in the other places separately with the task https://github.com/ballerina-platform/ballerina-lang/issues/40338.